.gitignore

```diff
@@ -4,3 +4,4 @@ npm-debug.log
 *.swp
 *.swo
 *.pyc
+yarn.lock
```

Comment:
I added this to ignore the yarn.lock on the local machine, the other alternative was to commit with the yarn.lock included.

I'd be highly interested in making a PR for the same. Here's what I think I could do:
1. Re-write package.json by running `yarn init` and `yarn add` keeping it as close as possible to the current npm one.
2. Run `yarn clean`, and add both yarn.lock and the file generated after cleanup to source control.
3. Commit and make a PR.
@526avijitgupta please propose any changes if you'd like.